src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -71,6 +72,12 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader->load('web.xml');
         $loader->load('services.xml');
+
+        if (PHP_VERSION_ID < 70000) {
```

Comment:
this will still trigger a deprecation warning for PHP 5.6 users. Do we want it ?